src/commands/image/imageSource/ContainerAppUpdateStep.ts

```diff
@@ -51,13 +55,25 @@ export class ContainerAppUpdateStep<T extends ImageSourceContext & IngressContex
             name: getContainerNameForImage(nonNullProp(context, 'image')),
         });
 
-        const updating = localize('updatingContainerApp', 'Updating container app...', containerApp.name);
-        progress.report({ message: updating });
-
-        await ext.state.runWithTemporaryDescription(containerApp.id, localize('updating', 'Updating...'), async () => {
-            context.containerApp = await updateContainerApp(context, context.subscription, containerAppEnvelope);
-            ext.state.notifyChildrenChanged(containerApp.managedEnvironmentId);
-        });
+        // Related: https://github.com/microsoft/vscode-azurecontainerapps/pull/805
```

Comment:
Did you mean to link the issue: https://github.com/microsoft/vscode-azurecontainerapps/issues/791?

I feel like that makes more sense but it's all linked so either is fine.

Reply:
Oh yeah I meant to link that, I'll fix real quick before merging

Reply:
Oh yeah I figured this PR description would get readers more quickly up to speed on why the retries were implemented